cmd/buildutil/info.go

```diff
@@ -397,7 +397,7 @@ func (u S3URL) Subpath(p ...string) S3URL {
 }
 
 func (u S3URL) String() string {
-	return fmt.Sprintf("s3://%s%s", u.Bucket, u.Key)
+	return fmt.Sprintf("s3://%s/%s", u.Bucket, u.Key)
```

Comment:
Why?

Reply:
That's what my Slack rant was about. The `Key` must not start with a slash as this is interpreted as a separate directory.